samza-core/src/main/java/org/apache/samza/clustermanager/ContainerProcessManager.java

```diff
@@ -186,7 +208,18 @@ public boolean shouldShutdown() {
 
   public void start() {
     log.info("Starting the container process manager");
-    metrics.start();
+
+    if (jvmMetrics != null) {
+      jvmMetrics.start();
```

Comment:
Why do 'metrics' need to be started?

Reply:
Because someone decided to have JVMetrics with their own executor pool